client/ayon_core/plugins/publish/integrate.py

```diff
@@ -464,6 +463,8 @@ def prepare_version(
             version_number,
             product_entity["id"],
             task_id=task_id,
+            author=instance.context.data.get("user"),
```

Comment:
@iLLiCiTiT we still need this to be passed as arg or will the user be collected implicitly (again, because `collect_current_ayon_user.py` does it already)?

Reply:
It should not be necessary. When service works as other user, server should auto-fill the value correctly.

Reply:
I'd prefer to be explicit tbh

Reply:
> I'd prefer to be explicit tbh

Well, version has `author` but product, representation and thumbnail has `createdBy` which is almost equivalent. I would rather prefer to not use the value as it might hide unresolved issues at other places.

Reply:
done at https://github.com/ynput/ayon-core/pull/615/commits/f6e3dcb4742e083664744de016476d8c134f5ef0

Reply:
BTW have you tested it?

Reply:
not your changes yet no, I'm still relying on my explicit `collect_current_ayon_user` of my branch, I can test it later this afternoon